fastlane/Fastfile

```diff
@@ -69,28 +76,27 @@ end
 desc 'Bump and commit app version and build number'
 private_lane :set_version_bump_build_and_commit do |options|
   if options[:platform] == 'tvOS'
-    increment_build_number_in_plist(VLC::infoPlistPath[:tvOS])
-    set_version_number_in_plist(VLC::infoPlistPath[:tvOS], options[:version])
+    increment_build_number_in_plist(VLC.info_plist_path[:tvOS])
+    set_version_number_in_plist(VLC.info_plist_path[:tvOS], options[:version])
   elsif options[:platform] == 'iOS'
-    increment_build_number_in_plist(VLC::infoPlistPath[:iOS])
-    increment_build_number_in_plist(VLC::infoPlistPath[:watchKitExtension])
-    increment_build_number_in_plist(VLC::infoPlistPath[:watchOS])
-    set_version_number_in_plist(VLC::infoPlistPath[:iOS], options[:version])
-    set_version_number_in_plist(VLC::infoPlistPath[:watchKitExtension], options[:version])
-    set_version_number_in_plist(VLC::infoPlistPath[:watchOS], options[:version])
+    increment_build_number_in_plist(VLC.info_plist_path[:iOS])
+    increment_build_number_in_plist(VLC.info_plist_path[:watchKitExtension])
+    increment_build_number_in_plist(VLC.info_plist_path[:watchOS])
+    set_version_number_in_plist(VLC.info_plist_path[:iOS], options[:version])
+    set_version_number_in_plist(VLC.info_plist_path[:watchKitExtension], options[:version])
+    set_version_number_in_plist(VLC.info_plist_path[:watchOS], options[:version])
   end
 
   commit_version_bump(message: 'Version Bump by fastlane', force: true)
 end
 
 desc 'Update changelog in iTunes Connect with the content from Docs/NEWS'
 private_lane :update_changelog do |options|
+  platform = options[:platform]
   # Splits the News by -------- get out the top notes
   changelog = File.read('../Docs/NEWS').split('-----------')[1].split('-----------').first
-  temp_changelog = changelog.split("${options[:platform]}")
-  if temp_changelog.count <= 1
-    temp_changelog = changelog.split("tvOS")
-  end
+  temp_changelog = changelog.split(platform)
+  temp_changelog = changelog.split('tvOS') if temp_changelog.count <= 1
```

Comment:
nice! I didn't know that's how you can write this

Reply:
Me neither tbh, rubocop made this change 😆